packages/common/src/index.ts

```diff
@@ -1389,3 +1390,24 @@ export const convertAdditionalMetric = ({
         metric: additionalMetric,
         tableLabel: table.label,
     });
+
+export const getTableMagicMetrics = (
+    table: CompiledTable,
+): Record<string, AdditionalMetric> =>
+    Object.values(table.dimensions).reduce((previous, dimension) => {
+        const entityName = extractEntityNameFromIdColumn(dimension.name);
+        if (entityName === null) {
+            return previous;
+        }
+        const magicMetric: AdditionalMetric = {
+            name: `${dimension.name}_count_distinct`,
+            label: `Count distinct ${dimension.label}`,
```

Comment:
When creating a custom metric manually, it says `Count distinct of ${dimension.label}`

![image](https://user-images.githubusercontent.com/1983672/167358281-4fe78f19-44f2-4978-91ff-578d88214756.png)